firefox-ios/WidgetKit/DownloadManager/DownloadLiveActivity.swift

```diff
@@ -5,9 +5,52 @@
 import WidgetKit
 import ActivityKit
 import SwiftUI
+import Foundation
 
 struct DownloadLiveActivityAttributes: ActivityAttributes {
     struct ContentState: Codable, Hashable {
+        struct Download: Codable, Hashable {
+            var id: UUID
+            var fileName: String
+            var mimeType: String
+            var hasContentEncoding: Bool?
+            var downloadPath: URL
+
+            var totalBytesExpected: Int64?
+            var bytesDownloaded: UInt64
```

Comment:
Lets make this Int64 instead to better match the Download class in client (since we'll be converting instances of that class to this struct)